### ISSUE TITLE
Simple Economics example

### ISSUE THREAD

**Issue by azev77** [thowell/IterativeLQR.jl#6]
Hi and thanks for this package!
Is it possible to apply your solver to a simple (linear quadratic) economics problem:
https://discourse.julialang.org/t/solving-the-4-quadrants-of-dynamic-optimization-problems-in-julia-help-wanted/73285

Btw, can it handle infinite horizon problems?

**Comment by thowell** [comment]
Yes, this package will work for economics problems that fit the iLQR problem formulation. However, it won't necessarily handle infinite horizon problems. You can approximate this with a large planning horizon.
